stable/cluster-autoscaler/values.yaml

```diff
@@ -51,7 +75,9 @@ rbac:
   ## If true, create & use RBAC resources
   ##
   create: false
-
+  ## If true, create & use Pod Security Policy resources
+  ## https://kubernetes.io/docs/concepts/policy/pod-security-policy/
+  pspEnabled: false
```

Comment:
I wouldn't put this under rbac.

Reply:
PSP requires RBAC and i'm adding additional RBAC policies to allow running autoscaler on the cluster where PSP is enabled